pkg/utils/file.go

```diff
@@ -133,3 +148,132 @@ func GetHomeDirectory() string {
 	}
 	return currentUser.HomeDir
 }
+
+func PrintDuDetails(files []DuDetails) {
+	for i, file := range files {
+		fmt.Printf("%d: %s size: %d mod date: %s\n", i, file.path, file.size, file.mtime.Format("2006-01-02T15:04:05"))
+	}
+}
```

Comment:
Is this used anywhere? Debug function? Should probably be removed or converted to log or return a string.

Reply:
removed as all of du functions.
not needed anymore